scripts/scp_api.py

```diff
@@ -558,7 +562,15 @@ def edit_study_description(
         if not accept_html and not self.is_valid_study_description(new_description):
             return {c_SUCCESS_RET_KEY: False, c_CODE_RET_KEY: c_INVALID_STUDY_DESC}
 
-        description_info = {"study_id": study_id, "description": new_description}
+        description_info = {
+            "study": {
+                "study_id": study_id,
+                "name": study_name,
```

Comment:
Unless the user wants to change the name of the study, it is not necessary include it in the `PATCH`.  Furthermore, it looks like the study name is only used to convert into a study ID.  I would remove this from the payload.

Reply:
makes sense, updated payload in `510b45b`